iphone/Maps/Classes/CarPlay/CarplayPlaceholderView.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+
+class CarplayPlaceholderView: UIView {
+  private let containerView = UIView()
+  private let imageView = UIImageView()
+  private let descriptionLabel = UILabel()
+  private let switchButton = UIButton(type: .system);
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    commonInit()
+  }
+
+  required init?(coder: NSCoder) {
+    super.init(coder: coder)
+    commonInit()
+  }
+
+  private func commonInit() {
+    backgroundColor = .clear
+
+    addSubview(containerView)
+
+    imageView.image = UIImage(named: "ic_carplay_activated")
+    imageView.contentMode = .scaleAspectFit
+    containerView.addSubview(imageView)
+
+    descriptionLabel.text = L("aa_used_on_the_car_screen")
+    descriptionLabel.font = UIFont.bold24()
+    descriptionLabel.textColor = UIColor.blackSecondaryText()
+    descriptionLabel.textAlignment = .center
+    descriptionLabel.numberOfLines = 0
+    containerView.addSubview(descriptionLabel)
+
+    switchButton.setTitle(L("aa_continue_on_the_phone"), for: .normal)
+    switchButton.addTarget(self, action: #selector(onSwitchButtonTap), for: .touchUpInside)
+    switchButton.setTitleColor(UIColor.whitePrimaryText(), for: .normal)
+    switchButton.titleLabel?.font = UIFont.medium16()
+    switchButton.titleLabel?.lineBreakMode = .byWordWrapping
+    switchButton.titleLabel?.textAlignment = .center
+    switchButton.backgroundColor = UIColor.linkBlue()
```

Comment:
I have tried to use `styleName = "FlatNormalButton"`, but was not happy with result:
1. Font seems too small
2. Pressed state is too transparent for this background?

I don't known if it worth to create separate style for one button, so I did styling manually without using themes system. Please let me know if you prefer to have it done in another way.

<img src="https://github.com/organicmaps/organicmaps/assets/1800899/b4c4344d-1aae-430d-a58c-f32ac971878c" width="200">

<img src="https://github.com/organicmaps/organicmaps/assets/1800899/f31cd146-152e-4e3d-8515-4340ff33ee55" width="200">

Reply:
@kirylkaveryn PTAL whenever you get a chance. Thanks!

Reply:
Looks nice and works well! Thanks!

I found that after reloading the scale is Ok now, but this bookmarks drawing changes and their amount increases as for another zoom level. This is not the part of your PR. Ill create an issue.